packages/gas-fee-controller/src/GasFeeController.ts

```diff
@@ -303,7 +303,7 @@ export class GasFeeController extends BaseControllerV2<
     getCurrentNetworkLegacyGasAPICompatibility: () => boolean;
     getCurrentAccountEIP1559Compatibility?: () => boolean;
     getChainId?: () => `0x${string}` | `${number}` | number;
-    getProvider: () => NetworkController['provider'];
```

Comment:
The `provider` property on NetworkController went away, so we can't use that as a type here. That said, I don't think this was right anyway, it's better to export a type from NetworkController.